.github/renovate.json5

```diff
@@ -78,6 +78,8 @@
       matchUpdateTypes: [
         'patch',
       ],
+      // See rust-lang/cargo#13546 and openssl/openssl#23376 for the exclusion
```

Comment:
End up using [`excludePackageNames`](https://docs.renovatebot.com/configuration-options/#excludepackagenames), though I completely don't know if it can detect lockfile-only updates.

Reply:
@epage any chance you know this is gonna work?

Reply:
RenovateBot will no try to upgrade us but if something else causes an upgrade, then it will still happen.

Granted, if they are running `cargo update <dep>` then nothing should update it.

Reply:
I don't know why they got upgraded by renovatebot. There was nothing preventing me from `cargo update openssl --precise <oldversion>`. Weird.